kcauto-kai.sikuli/repair.sikuli/repair.py

```diff
@@ -254,11 +254,15 @@ def _update_combat_next_sortie_time(self):
         """Method to update the Combat module's next sortie timer based on the
         shortest timer in the internal repair timers list.
         """
-        self.repair_timers.sort()
-        shortest_timer = self.repair_timers[0]
+        if len(self.repair_timers) > 0:
+            if len(self.repair_timers) > 1:
+                self.repair_timers.sort()
+            shortest_timer = self.repair_timers[0]
+        else:
+            shortest_timer = datetime.now()
 
-        if shortest_timer > self.combat.next_combat_time:
+        if shortest_timer < self.combat.next_combat_time:
             self.combat.next_combat_time = shortest_timer + timedelta(
                 minutes=1)
             Util.log_msg("Delaying next combat sortie to {}".format(
-                self.combat.next_combat_time.strftime('%Y-%m-%d %H:%M:%S')))
+                self.combat.next_combat_time.strftime('%Y-%m-%d %H:%M:%S')))
```

Comment:
Add newline at end of file (see error W292 in the [PEP8 styleguide](http://pep8.readthedocs.io/en/latest/intro.html)).